src/sentry/workflow_engine/models/detector.py

```diff
@@ -26,6 +26,7 @@ class Detector(DefaultFieldsModel, OwnerModel):
         unique=True,
         on_delete=models.SET_NULL,
     )
+    type = models.CharField(max_length=200)
```

Comment:
This is safe to add without a default because this model is completely unused so far.

Reply:
🤞 that ci is okay with this 😅

Reply:
I disabled checks on the migration so it's fine, but don't follow my example here lol